bin/geni-sync-wireless

```diff
@@ -0,0 +1,618 @@
+        self.ensure_project_members_exist()
+
+        # Make sure all wimax-enabled projects exist as orbit groups
+        self.ensure_projects_exist()
+
+        # Make sure all orbit users are in proper wimax group
+        self.ensure_project_members_in_groups()
+
+        # Make sure the admins of orbit groups match the leads of GENI projects
+        self.ensure_project_leads_are_group_admins()
+
+        # If we're doing cleanup, 
+        #   delete group members who aren't project members
+        #   delete groups that aren't GENI projects
+        #   disable any users not in any GENI project
+        if self._cleanup:
+            self.delete_group_members_not_in_project()
+            self.delete_groups_without_projects()
+            self.disable_users_in_no_project()
+
+    # Make sure that the holdingpen group exists
+    def ensure_holdingpen_group_and_admin(self):
+
+        # Find the holdingpen admin among the GENI members read
+        holdingpen_admin_info = None
+        for member_id, member_info in self._geni_members.items():
+            if member_info['username'] == self._options.holdingpen_admin:
+                holdingpen_admin_info = member_info
+                break
+
+        if not holdingpen_admin_info:
+            self.error("Holdingpen admin not in GENI: %s" % \
+                           self._options.holdingpen_admin)
+
+        # Grab 'pretty name' for holdingpen admin
+        admin_pretty_name = self.get_pretty_name(holdingpen_admin_info)
+        holdingpen_admin_username = \
+            self.to_orbit_name(self._options.holdingpen_admin)
+        holdingpen_admin_ssh_keys = holdingpen_admin_info['ssh_keys']
+
+        ldif_text = ""
+        if self._options.holdingpen_group not in self._orbit_groups:
+            ldif_text = ldif_text + \
+                self._orb.ldif_for_group(self._options.holdingpen_group,
+                                   self.holdingpen_group_description)
+            ldif_text = ldif_text + \
+                self._orb.ldif_for_group_admin(self._options.holdingpen_group,
+                                         holdingpen_admin_username,
+                                         self._options.holdingpen_group)
+            print "Creating holdingpen group: %s" % \
+                self._options.holdingpen_group
+
+        if holdingpen_admin_username not in self._orbit_users:
+            ldif_text = ldif_text + \
+                self._orb.ldif_for_user(holdingpen_admin_username,
+                                  self._options.holdingpen_group,
+                                  admin_pretty_name,
+                                  holdingpen_admin_info['first_name'],
+                                  holdingpen_admin_info['email_address'],
+                                  holdingpen_admin_info['last_name'],
+                                  holdingpen_admin_ssh_keys,
+                                  self.holdingpen_group_description, None)
```

Comment:
If both blocks of LDIF (group and admin) are in the same transaction, it is happy either way. Otherwise we get a chicken-and-egg either way we order it.

Roger about irods username.